packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -197,9 +205,9 @@ export interface IContextualMenuItem {
   href?: string;
 
   /**
-   * A collection of submenu items
+   * A collection of submenu items, or an IContextualMenuProps object to apply for the sub menu
    */
-  items?: IContextualMenuItem[];
+  items?: IContextualMenuItem[] | IContextualMenuProps;
 
```

Comment:
Maybe we should make a submenu property that can take in Icontextualmenuitems[] | Icontextualmenuprops. This sets up the expectation that the contextualmenu item is the owner of a submenu which may just include items, in which case the submenu is autogenerated, otherwise it gets customized. I know that seems exactly the same but having it named like that makes it more clear for me. Having items that could also be a contextualmenu seems really confusing.